packages/api-react/src/services/farmer.ts

```diff
@@ -231,25 +239,75 @@ export const farmerApi = apiWithTag.injectEndpoints({
               { type: 'Pools', id: 'LIST' },
             ]
           : [{ type: 'Pools', id: 'LIST' }],
+      onCacheEntryAdded: onCacheEntryAddedInvalidate(baseQuery, api, [
+        {
+          command: 'onSubmittedPartial',
+          service: Farmer,
+          endpoint: 'getPoolState',
+        },
+        {
+          command: 'onFailedPartial',
+          service: Farmer,
+          endpoint: 'getPoolState',
+        },
+      ]),
+    }),
+
+    getPartialStatsOffset: query(build, Farmer, 'getPartialStatsOffset', {
+      providesTags: ['partialStats'],
+    }),
+
+    resetPartialStats: mutation(build, Farmer, 'resetPartialStats', {
+      invalidatesTags: ['partialStats'],
     }),
 
     setPayoutInstructions: mutation(build, Farmer, 'setPayoutInstructions', {
       invalidatesTags: (_result, _error, { launcherId }) => [{ type: 'PayoutInstructions', id: launcherId }],
     }),
 
-    getFarmingInfo: query(build, Farmer, 'getFarmingInfo', {
+    getNewFarmingInfo: query(build, Farmer, 'getNewFarmingInfo', {
```

Comment:
We should consider whether "new" is appropriate to include in any API names. If we keep "new" in the names, we should have a plan to update them at some point.

Reply:
Nevermind, I see it's an existing name.